ods_ci/utils/scripts/testconfig/test-variables.yml

```diff
@@ -81,3 +81,4 @@ APPLICATIONS_NAMESPACE: APPLICATIONS_NAMESPACE
 MONITORING_NAMESPACE: MONITORING_NAMESPACE
 OPERATOR_NAMESPACE: OPERATOR_NAMESPACE
 NOTEBOOKS_NAMESPACE: NOTEBOOKS_NAMESPACE
+OPENSHIFT_PIPELINES_VERSION: latest
```

Comment:
i think it'd better to follow the format of the other keys `OPENSHIFT_PIPELINES_VERSION: OPENSHIFT_PIPELINES_VERSION`